nautobot/extras/tests/test_jobs.py

```diff
@@ -18,34 +17,27 @@
 from nautobot.extras.jobs import get_job, run_job
 from nautobot.extras.models import FileProxy, JobResult, Status, CustomField
 from nautobot.extras.models.models import JobLogEntry
-from nautobot.utilities.testing import CeleryTestCase, TestCase
+from nautobot.utilities.testing import CeleryTestCase, TransactionTestCase
 
 
 # Use the proper swappable User model
 User = get_user_model()
 
 
-# Override the JOB_LOGS to None so that the Log Objects are created in the default database.
-# This change is required as JOB_LOGS is a `fake` database pointed at the default. The django
-# database cleanup will fail and cause tests to fail as this is not a real database.
-@mock.patch("nautobot.extras.models.models.JOB_LOGS", None)
-class JobTest(TestCase):
+class JobTest(TransactionTestCase):
     """
     Test basic jobs to ensure importing works.
     """
 
+    databases = ("default", "job_logs")
     maxDiff = None
 
-    @classmethod
-    def setUpTestData(cls):
-        cls.job_content_type = ContentType.objects.get(app_label="extras", model="job")
-
     def setUp(self):
-        super().setUp()
-
+        self.job_content_type = ContentType.objects.get(app_label="extras", model="job")
```

Comment:
Did you mean to exclude `super().setUp()` here?

Reply:
It's not needed currently as our `TransactionTestCase`, unlike our `TestCase`, currently has no base `setUp` method defined. I can add it back in for the eventual possibility that it might though.